devel/tbb/Portfile

```diff
@@ -29,7 +28,14 @@ checksums           rmd160  64fd69e42540ef8968d9a3e36cb23816ca9de647 \
 use_configure       no
 
 # Force a compatible compiler
-compiler.blacklist-append *gcc* {clang < 602} {macports-clang-3.[0-7]}
+compiler.blacklist-append \
+                    {clang < 602} {macports-clang-3.[0-7]}
+
+# Not sure why it was disabled at all, but at least do it conditionally.
+platform darwin i386 {
+    compiler.blacklist-append \
+                    *gcc*
+}
```

Comment:
I don't understand this; there wasn't anything about i386 / gcc before.

Reply:
@pmetzger Line 32 had `*gcc*` in blacklist.

Reply:
@pmetzger If you prefer just dropping it from blacklist, let’s do that. It is not picked by default anywhere aside of powerpc anyway, and arguably there is little sense to prohibit it, if someone wants to use it as a custom option.

Reply:
@pmetzger I removed that chunk, it is redundant anyway.